.github/workflows/deploy.yml

```diff
@@ -23,6 +23,8 @@ jobs:
         DEVO_AWS_SECRET_ACCESS_KEY: ${{secrets.DEVO_AWS_SECRET_ACCESS_KEY}}
         AWS_ACCESS_KEY_ID: ${{secrets.AWS_ACCESS_KEY}}
         AWS_SECRET_ACCESS_KEY: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+        BETA_EVENT_INGESTION_URL: ${{secrets.BETA_EVENT_INGESTION_URL}}
```

Comment:
init: This needs to remove?

Reply:
Yup removed from here and the canary deploy script as well. Cherry picked hence dint see this change to remove later. Good catch!